test/augmentation/test_container.py

```diff
@@ -357,6 +357,18 @@ def test_random_erasing(self, device, dtype):
         out = aug(input, input)
         assert torch.all(out[1][out[0] == fill_value] == 0.0)
 
+    def test_resize(self, device, dtype):
+        size = 50
+        input = torch.randn(3, 3, 100, 100, device=device, dtype=dtype)
+        mask = torch.randn(3, 1, 100, 100, device=device, dtype=dtype)
+        aug = K.AugmentationSequential(K.Resize((size, size), p=1.0), data_keys=["input", "mask"])
```

Comment:
Maybe test with explicit interpolation mode? -- to cover the issue

Reply:
It is already added as `extra_args` when handling the mask data key (and the combination of the extra args added is the issue) so it is covering the exact problem.

Reply:
yeap, I had commented because coverage is mad with the case 😅